src/map/mob.cpp

```diff
@@ -2459,14 +2474,6 @@ void mob_damage(struct mob_data *md, struct block_list *src, int damage)
 int mob_getdroprate(struct block_list *src, std::shared_ptr<s_mob_db> mob, int base_rate, int drop_modifier)
 {
 	int drop_rate = base_rate;
-
-	if (battle_config.mob_size_influence) { // Change drops depending on monsters size [Valaris]
```

Comment:
I guess I'm curious why it was removed from here since any other call to this function will also have to make sure to include the size modifier manually now. Seems more logical to keep this check with the special_state fix in `mob_getdroprate`.

Reply:
Yep let's just add a parameter

Reply:
@aleos89:
It was most likely removed when the atcommand was added that does not have access to any specific spawned monster.

@Singe-Horizontal:
Could you maybe hand over the actual mob_data* to the function? Just in case some other checks will be necessary in the future or if someone wants to hook with a source modification to that function. Also please make sure to revert the empty line removals and style changes wherever possible please.
Thank you!

Ps:
All these assumptions with SZ_MEDIUM actually being SZ_SMALL make me wanna puke btw... XD Just because someone was too lazy to initialize this properly.

Reply:
The forgotten curse of the size enum